input/drivers/switch_input.c

```diff
@@ -122,50 +192,62 @@ static void switch_input_poll(void *data)
       }
    }
 
-   if (hidMouseButtonsHeld() & MOUSE_LEFT)
-   {
-      sw->mouse_button_left = true;
-   }
-   else
+   // update physical mouse buttons only when they change
+   // this allows the physical mouse and touch mouse to coexist
+   uint64_t mouse_current_report = hidMouseButtonsHeld();
+   if ((mouse_current_report & MOUSE_LEFT) != (sw->mouse_previous_report & MOUSE_LEFT))
    {
-      sw->mouse_button_left = false;
+      if (mouse_current_report & MOUSE_LEFT)
+      {
```

Comment:
When a code block is only one line, then drop the brackets around it.

Reply:
all fixed in latest force push